src/libraries/System.Net.Security/tests/FunctionalTests/SslAuthenticationOptionsTest.cs

```diff
@@ -65,7 +66,8 @@ public async Task ClientOptions_ServerOptions_NotMutatedDuringAuthentication()
                         EncryptionPolicy = clientEncryption,
                         LocalCertificateSelectionCallback = clientLocalCallback,
                         RemoteCertificateValidationCallback = clientRemoteCallback,
-                        TargetHost = clientHost
+                        TargetHost = clientHost,
+                        ValidationPolicy = policy
```

Comment:
Personally, I always end multi-line things like this with a comma so the spurious edit of the last line doesn't show up later.